### PR TITLE
feat: add li to ul

### PR DESCRIPTION
**Pull request by Numabeharuka**
# [Issue No.01](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#dom%E6%A7%8B%E7%AF%89)  
1.このDOMをhtml内のulの中に差し込んでください  
`<li>これです</li>`
## [CodeSandBox](https://codesandbox.io/s/exciting-http-unwoq?file=/script.js)
Latest commit: [`ced2789`](https://github.com/Numabeharuka/Lesson-js/commit/ced27892bfbdb5a76dc03b2d4f380ae9381b2d4b)  
### Anxious areas  
- I'm not sure if js is correct.
- I'm not sure if I'm doing Git and GitHub correctly.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+const li_element = document.createElement("li");
+li_element.textContent = "これです";
```
Comment (kenmori) [inline script.js:4]: jsでは、あまりスネークケースを変数名にしません。何か参考にしてそうしましたか？もしそうならそのリンクを教えてください。
```suggestion
liElement.textContent = "これです";
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (kanakonagiri) [inline index.html:2]:
```suggestion
<html lang="ja">
```
I think it's better to change this part.

**Review by kanakonagiri — COMMENTED** [review]
> I'm not sure if I'm doing Git and GitHub correctly.

I think how to put out PR and how to use GitHub is good!

File: script.js (hunk 1 above)
Comment (Numabeharuka) [reply script.js]: スネークケースについては特に参考にしていません。申し訳ございません。classのように記述してしいまいました。

File: index.html (hunk 2 above)
Comment (Numabeharuka) [reply index.html]: Thank you for pointing this out.
We will change it.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+const lielement = document.createElement("li");
+lielement.textContent = "これです";
+const ulelement = document.querySelector("ul");
+ulelement.appendChild(lielement);
```
Comment (csakyo) [inline script.js:6]: I think the camel case is the way to go here. Take a close look at the code your teacher wrote.
If you want, you can refer to it.

https://designsupply-web.com/media/development/4052/

Comment (Numabeharuka) [reply script.js:6]: I overlooked the capitalization part.
Thank you very much. I will refer to it.

**Review by kenmori — APPROVED** [review]

**Review by csakyo — APPROVED** [review]

**Pull request merged**